src/Main.java

```diff
@@ -0,0 +1,31 @@
+import javax.swing.*;
```

Comment:
Try always to import only the classes that you need instead of loading the whole swing package